src/integration-test/java/com/mongodb/jdbc/integration/MongoSQLIntegrationTest.java

```diff
@@ -55,4 +77,70 @@ public void loadTestConfigs() throws IOException {
         }
         return dynamicTests;
     }
+
+    @Test
+    public void testTypesOther() throws SQLException, IOException, IllegalAccessException {
+        DataLoader loader = new DataLoader(TEST_DATA_OTHER_DIRECTORY);
+        loader.dropCollections();
```

Comment:
Why do we need to drop collections at the start of this test?

Reply:
I added that in case the types_other collection already exists from running this test previously on an existing local ADL, in this case it drops only the types_other collection as right now that's the only collection in the TEST_DATA_OTHER_DIRECTORY directory.

Reply:
Ok I see. This may become a non-issue if we manage to work out the ideas in other comments (re: using extended json to write this data in yaml data files like the other data).

Reply:
I've removed the dataloader calls here in testTypesOther().  As you mention, it was worked out by using the extended json to load the data.